aspnetcore/fundamentals/http-requests/samples/3.x/HttpClientFactorySample/GitHub/GitHubService.cs

```diff
@@ -33,10 +34,12 @@ public async Task<IEnumerable<GitHubIssue>> GetAspNetDocsIssues()
 
             response.EnsureSuccessStatusCode();
 
-            var result = await response.Content
-                .ReadAsAsync<IEnumerable<GitHubIssue>>();
-
-            return result;
+            using (var responseStream =
+                    await response.Content.ReadAsStreamAsync())
+            {
+                return await JsonSerializer.DeserializeAsync
+                    <IEnumerable<GitHubIssue>>(responseStream);
+            }
```

Comment:
This is the general pattern that's used now. It appears to work well and isn't _too_ verbose. We could use a `using` declaration now we've got C# 8 but I haven't really seen much of that in use in the docs so far.

Reply:
I recommend using the C# 8 `using` declaration in the 3.x examples. It's not widely adopted in the docs simply because we haven't had the time.

Reply:
Thanks @scottaddie. That's done.